src/css/flexboxgrid.css

```diff
@@ -30,50 +30,32 @@
   padding-left: var(--outer-margin, 2rem);
 }
 
-.row {
+.row, .col {
   box-sizing: border-box;
   display: flex;
   flex: 0 1 auto;
-  flex-direction: row;
   flex-wrap: wrap;
   margin-right: var(--gutter-compensation, -0.5rem);
   margin-left: var(--gutter-compensation, -0.5rem);
 }
 
+.row {
+  flex-direction: row;
+}
+
 .row.reverse {
   flex-direction: row-reverse;
 }
 
+.col {
+  flex-direction: column;
+}
+
 .col.reverse {
   flex-direction: column-reverse;
 }
 
-.col-xs,
-.col-xs-1,
-.col-xs-2,
-.col-xs-3,
-.col-xs-4,
-.col-xs-5,
-.col-xs-6,
-.col-xs-7,
-.col-xs-8,
-.col-xs-9,
-.col-xs-10,
-.col-xs-11,
-.col-xs-12,
-.col-xs-offset-0,
-.col-xs-offset-1,
-.col-xs-offset-2,
-.col-xs-offset-3,
-.col-xs-offset-4,
-.col-xs-offset-5,
-.col-xs-offset-6,
-.col-xs-offset-7,
-.col-xs-offset-8,
-.col-xs-offset-9,
-.col-xs-offset-10,
-.col-xs-offset-11,
-.col-xs-offset-12 {
+[class|="col"] {
```

Comment:
That is not gonna work.
`|=` means class attribute `starts` with `col`.
What if a developer will add additional class in the beginning of the attribute value? Like `<div class="something col-sm-..."`?

I would use something like `[class*="col-xs]..., [class*="col-sm]`....
But again, even that approach restricted a developer on adding some utils classes like `col-xs-whatever`.

Reply:
About first one - good point. Didn't think about it for some reason. Gonna fix now.

> But again, even that approach restricted a developer on adding some utils classes
Well, it doesn't, actually.
`[class*="col-"]` would clearly cut it for both usecases, since it would match both the usual col classes and custom utility ones too:
```css
<element class="col-xs-6 col-xs-make-it-very-cool col-lg-make-it-very-cool">
```

In any case, current waterfall of classes every 100 or so lines does not cut it for the second use-case either,  so the only drawback to defining common rules as `[class*="col-"]` would be that all similarly-looking classes would fit this selector (which I personally view more like a nice feature).